### PR TITLE
env var thonny version

### PR DESCRIPTION
**Pull request by kr-g**
this belongs to 
https://github.com/thonny/thonny/issues/2492

### CODE REVIEW

**Comment by kr-g** [comment]
@aivarannamaa 
please review, and pull if requirement is fine with you

**Comment by kr-g** [comment]
thonny `get_version()` method is used at several places, so i changed the handling to return a cached value rather then using a global variable (as proposed with with my initial change)

references 

```
./thonny/__init__.py:def get_version():
./thonny/__init__.py:    main_logger.info("Thonny version: %s", get_version())
./thonny/assistance.py:            "thonny_version": thonny.get_version(),
./thonny/plugins/pip_gui.py:                if req.project_name == "thonny" and thonny.get_version() not in req:
./thonny/plugins/about.py:            self.main_frame, text="Thonny " + thonny.get_version(), font=heading_font
./thonny/plugins/cpython_ssh/cps_back.py:        return f"/tmp/thonny-backend-{thonny.get_version()}-{self._user}"
./thonny/plugins/cpython_ssh/cps_back.py:        if self._get_stat_mode_for_upload(launch_dir) and not thonny.get_version().endswith("-dev"):
./thonny/running.py:    get_version,
./thonny/running.py:        env["THONNY_VERSION"] = get_version()

```

File: thonny/__init__.py [hunk 1]
```diff
@@ -143,11 +143,19 @@ def is_portable():
         return abs_location.startswith("/media/") or abs_location.startswith("/mnt/")
 
 
+_THONNY_VERSION = None
+
+
 def get_version():
+    global _THONNY_VERSION
+    if _THONNY_VERSION:
+        # return a copy
```
Comment (aivarannamaa) [inline thonny/__init__.py:152]: Why copy?

**Comment by aivarannamaa** [comment]
Thanks! Looks good now.

**Pull request merged**